### ISSUE TITLE
Startup too slow [Suggestion]

### ISSUE THREAD

**Issue by williamgh2019** [Eugeny/tabby#6253]
Version: 1.0.176
Platform: macOS x64 21.4.0
Plugins: none
Frontend: xterm

It could be better if we can startup Tabby faster like Mac Terminal.

**Comment by MANICX100** [comment]
Agreed, but you must understand Tabby is a larger application than the built in terminal prompt both in features and memory usage.

Personally I use quake mode (quick summon) to have it instantly appear and never close it so the slow initial startup doesn't affect me.

You will be prompted to use ctrl and space on first launch, you can get this hotkey under settings, config.

**Comment by TisnKu** [comment]
Opening new tabs and closing tabs are slow. When tab is being opened or closed, there is this animation of title collapse/extend which looks laggy. Maybe we can remove the animation?

**Comment by amilor** [comment]
I have noticed this lagginess when opening/closing tabs after I updated to 1.0.176. Disabled it by turning off the Settings > Application > Accessibility > Enable animations toggle.

**Comment by williamgh2019** [comment]
> Agreed, but you must understand Tabby is a larger application than the built in terminal prompt both in features and memory usage.
> 
> Personally I use quake mode (quick summon) to have it instantly appear and never close it so the slow initial startup doesn't affect me.
> 
> You will be prompted to use ctrl and space on first launch, you can get this hotkey under settings, config.

If you compare startup with Termius SSH you will see, Tabby startup is really slow and Termius start very fast. If Tabby can fix this issue it could be better.
